test/destory.test.js

```diff
@@ -0,0 +1,26 @@
+var fs = require('fs');
+var path = require('path');
+
+var test = require('tape');
+var pixelmatch = require('pixelmatch');
+
+var hijack = require('./hijack');
+var byebyte = require('../index');
+
+var width = 400;
+var height = 198;
+
+test('[destroy] basic opts', function(assert) {
+    hijack.hijack();
```

Comment:
Rather than executing a side effect as a way to force the function to return something, what do you think about having the `destroy` and `shuffle` modules' `fn` functions accept a 3rd parameter which is a `getRandomInt` function?

Then you could remove the `hijack` module's `util` stubbing functionality and just rely on its `predictable` method directly.

So it might look like this in the `fn` functions:

```
    fn: function(fileBuffer, opts, getRandomInt) {
        var randomIntFnc = getRandomInt || util.getRandomInt;

        // ...
        var offset = randomIntFnc(start, stop);
        // ...
    }
```

Then you can pass in `predictable` fairly easily in tests:

```
test('[destroy] basic opts', function(assert) {
    // ...
    var out = byebyte.destroy(fileBuffer, {min:.3, max:.5}, hijack.predictable());
});
```